test/unit/util/support.js

```diff
@@ -5,7 +5,7 @@ describe('util/support', () => {
     // We take a copy of a known native property so that we can mock how our code will behave when
     // it finds a native implementation of a function. The polyfilled property just needs to match
     // what a non-native property looks like on an element (must have a `value` function)
-    const nativeProperty = Object.getOwnPropertyDescriptor(Element.prototype, 'getAttribute');
+    const nativeProperty = Object.getOwnPropertyDescriptor(Function.prototype, 'toString');
```

Comment:
If we're using webcomponents to polyfill ShadowDOM, then everything gets wrapped because they're re-assigning the prototype for everything starting from the EventTarget.
Because of this attempt to get a native property from the Element results in a property that is not native at all but just a wrapper-function.